.github/workflows/push.yml

```diff
@@ -0,0 +1,141 @@
+name: GitHub Actions
+on: [ push ]
+concurrency: ci-${{ github.ref }} # to avoid tag collisions in the ECR
+env:
+  # Name of the image in the ECR
+  APP_IMAGE: keboola-component
+
+  # Developer portal login
+  KBC_DEVELOPERPORTAL_VENDOR: "keboola"
+  KBC_DEVELOPERPORTAL_APP: "keboola.synapse-transformation"
+  KBC_DEVELOPERPORTAL_USERNAME: "keboola+synapse_transformation_github_actions"
+  KBC_DEVELOPERPORTAL_PASSWORD: ${{ secrets.KBC_DEVELOPERPORTAL_PASSWORD }}
+
+  # DockerHub login
+  DOCKERHUB_USER: "keboolabot"
```

Comment:
na tohle je myslím, taky organization secret, ale je to asi jedno :)

Reply:
já ho tady nedávám, aby bylo vidět který uživatel je použitý